README.md

```diff
@@ -40,6 +40,20 @@ the fly:
 This can be placed in a background thread, or triggered by some user
 request.
 
+### Declaring Dependencies to Static Resources
+
+Some Clojure libraries, such as [HugSQL](https://www.hugsql.org/#using-def-db-fns),
+define functions in a namespace based on the content of a static resource file.
+ns-tracker is able to reload such a namespace when the resource file is modified
+with the help of a bit metadata.
+
+You will need to declare the resources under the `:ns-tracker/resource-deps`
+key in the namespace's metadata:
+
+    (ns ^{:ns-tracker/resource-deps ["sql/queries.sql"]} example.db)    
```

Comment:
I think we should prefer an example that uses the `attr-map?` argument of `ns`. Perhaps something like:

```
(ns example.db
  {:ns-tracker/resource-deps ["sql/queries.sql"]}
  (:require example.utils))
```